viewer_clients/cmdline_client/cmd_line_client.py

```diff
@@ -113,6 +113,25 @@ def check_run_names(client, check_names):
 
     return run_info
 
+def add_filter_conditions(report_filter, filter_str):
+    '''This function fills some attributes of the given report filter based on
+    the filter string which is provided in the command line. The filter string
+    has to contain three parts divided by colons: the severity, checker id and
+    the file path respectively. The file path can contain joker characters, and
+    the checker id doesn't have to be complete (e.g. unix).'''
+
+    if filter_str.count(':') != 2:
+        print 'Filter string has to contain two colons.'
+        sys.exit(1)
+
+    (severity, checker, path) = filter_str.split(':')
```

Comment:
What is there are spaces in the given filter: ```" high: unix : main.cpp  "```
It might be useful to strip the values?